README.md

```diff
@@ -33,6 +33,23 @@ In case of error try
 service netfilter-persistent save  
 service netfilter-persistent reload
 ```
+Now, install dependencies using `glide`.
+```
+export PATH=$PATH:$GOPATH/bin
+curl https://glide.sh/get | sh
+glide install
```

Comment:
Installing glide so that dependencies could be installed from this. Just like we discussed on the issue.